can-value.md

```diff
@@ -38,61 +38,37 @@
 At its simplest, [can-value.with] can be used to
 create an observable from another initial value:
```

Comment:
We should also include some introduction to the example.  Examples should try as much as possible to have real-world names/ideas.  For example, 

> The following shows creating an `age` observable, reading it's value, and then listening to when the page changes:

```js
import {value, Reflect as canReflect} from "can";

const age = value.with(15);

console.log( age.value ); //-> 15

canReflect.onValue(age, function(newValue) {
  console.log( newValue ); //-> 16 ... time to drive!
});

age.value = 16;
```